crypto/keyring/doc.go

```diff
@@ -1,9 +1,9 @@
-// Package keys provides common key management API.
+// Package keyring provides common key management API.
```

Comment:
this doc was originally written when the package was named `keys` but the package has since been renamed

Ref: https://github.com/cosmos/cosmos-sdk/pull/5833